tests/console/389ds/tls_389ds_server.pm

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Implement & Integrate 389ds + sssd test case into openQA,
```

Comment:
Need note somewhere that this case now only supported to be run for migration path from SLES15SP3 to SLES15SP4.

Reply:
I think it depended on your cases. I can add comment to this condition, but I don't like to add code to this condition.